package.json

```diff
@@ -23,7 +23,7 @@
         "prettier-watch": "onchange './**/*.{ts,tsx}' -- prettier --write {{changed}}",
         "lint": "eslint . --ext .ts,.tsx",
         "lint:fix": "eslint . --ext .ts,.tsx --fix",
-        "ts-build": "tsc -b tsconfig.build.json && npm run postbuild -ws --if-present",
+        "ts-build": "tsc -b tsconfig.build.json && npm run -w nango postbuild",
```

Comment:
is it not the same?

Reply:
yeah it is, I just want to clarify it's a one of and the other packages should not rely on it